openstack/maia/alerts/maia-prom.alerts

```diff
@@ -0,0 +1,36 @@
+groups:
+- name: maia-prom.alerts
+  rules:
+  - alert: MaiaPromFederationDown
+    expr: |
+        up{job=~"prometheus-openstack|
+            prometheus-infra-collector|
+            prometheus-vmware|
+            cronus-reputation-statistics"} < 1
```

Comment:
I would strip this in dedicated alerts. Otherwise it always invokes the second check, which service is actually down. They are getting evaluated anyway, so no need to combine it.